assembly-combined-package/config/linkis-env.sh

```diff
@@ -48,9 +48,9 @@ ENGINECONN_ROOT_PATH=/appcom/tmp
 #RESULT_SET_ROOT_PATH=hdfs:///tmp/linkis ##hdfs:// required
 
 ### Provide the DB information of Hive metadata database.
-HIVE_META_URL=
-HIVE_META_USER=
-HIVE_META_PASSWORD=
+HIVE_META_URL=""
```

Comment:
It is recommended to add comments. If there are special characters, they need to be enclosed in quotation marks

Reply:
> It is recommended to add comments. If there are special characters, they need to be enclosed in quotation marks

good idea!

Reply:
good idea!